src/data/text/item_descriptions.h

```diff
@@ -1123,147 +1123,147 @@ static const u8 sLeftoversDesc[] = _(
 
 static const u8 sDragonScaleDesc[] = _(
     "A strange scale\n"
-    "held by DRAGON-\n"
-    "type POKéMON.");
+    "held by Dragon-\n"
+    "type Pokémon.");
 
 static const u8 sOvalStoneDesc[] = _(
     "Helps a certain\n"
-    "POKéMON to evolve.\n"
+    "Pokémon to evolve.\n"
     "Looks like an egg.");
 
 static const u8 sProtectorDesc[] = _(
-    "A certain POKéMON\n"
+    "A certain Pokémon\n"
     "loves it. It's quite\n"
     "heavy.");
 
 static const u8 sElectirizerDesc[] = _(
-    "A certain POKéMON\n"
+    "A certain Pokémon\n"
     "loves it. It's full\n"
     "of electricity.");
 
 static const u8 sMagmarizerDesc[] = _(
-    "A certain POKéMON\n"
+    "A certain Pokémon\n"
     "loves it. It's full\n"
     "of magma energy.");
 
 static const u8 sDubiousDiscDesc[] = _(
-    "A certain POKéMON\n"
+    "A certain Pokémon\n"
     "loves it. It's over-\n"
     "flowing with data.");
 
 static const u8 sReaperClothDesc[] = _(
     "Loved by a certain\n"
-    "POKéMON. Imbued\n"
+    "Pokémon. Imbued\n"
```

Comment:
Should probably be "A certain Pokémon\nloves it." for consistency

Reply:
Turns out "Loved by a certain\nPokémon." is the language used in the main series, so I've changed the other items rather than the Reaper Cloth.